src/lib/EventSubscriber/AppRendererSubscriber.php

```diff
@@ -51,7 +51,7 @@ public function renderApp(FilterResponseEvent $event)
 
         $response = $event->getResponse();
 
-        if ($response->isRedirect()) {
+        if ($response->isRedirect() || $response instanceof NoRenderResponse) {
```

Comment:
> There will be other Response classes that use this
Which ones ?

I don't see the problem with it being in the parent. That is what inheritance is for.
The App doesn't render the App structure when the Response is a redirect. The `ResetResponse` acts like a redirect, but isn't really one because of frontend requirements. 

I'd like to insist on that test :)